examples/ping.rs

```diff
@@ -82,3 +75,13 @@ async fn main() -> Result<(), Box<dyn Error>> {
         }
     }
 }
+
+/// Our network behaviour.
+///
+/// For illustrative purposes, this includes the `KeepAlive` behaviour so a continuous sequence of
```

Comment:
```suggestion
/// For illustrative purposes, this includes the [`KeepAlive`](behaviour::KeepAlive) behaviour so a continuous sequence of
```

To make sure this doesn't become outdated in the future.

Reply:
I don't think docs run on examples but we can still do this :)